tests/testthat/test-finalsize_by_group.R

```diff
@@ -0,0 +1,292 @@
+test_that("Check final size calculation is correct in simple case", {
+  r0 <- 2
+  contact_matrix <- matrix(r0 / 200.0, 2, 2)
+  demography_vector <- rep(100.0, 2) |> as.matrix()
+  psusc <- rep(1.0, 2) |> as.matrix()
+  susc <- rep(1.0, 2) |> as.matrix()
+
+  contact_matrix = apply(
```

Comment:
The contact is already correctly scaled above, so not needed to do here again

Reply:
Removed redundant scaling in https://github.com/epiverse-trace/finalsize/pull/39/commits/ba782bc3603e38067babc3c652c5427ebfb119f0